CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # CHANGELOG
 
+### 4.0.0
+
+* `relativePaths` is now automatically set to `false` when `publicPath` is used
+* `publicPath` now uses `webpack.config.js`'s `output.publicPath` by default (when not set in `OfflinePlugin` itself)
+* Added `AppCache.output` which is replacement for `AppCache.directory`. The last is deprecated now
+* Added `ServiceWorker.publicLocation` and `AppCache.publicLocation`
```

Comment:
Sure. I thought I added it to README.. Path is the most complicated thing with this plugin, especially when there should be a reasonable default. I think I need to write a doc about this options and paths in general. Small note won't be sufficient :-(